code/citadel/cit_guns.dm

```diff
@@ -192,10 +192,10 @@
 /datum/design/foam_x9
```

Comment:
at this point just move all of the foam gun designs to autoylathe unless there's a reason to not move it
@deathride58 @Poojawa thoughts?